_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,227 @@
+---
```

Comment:
Big problem with this rule. See https://github.com/act-rules/act-rules.github.io/pull/1412#issuecomment-733606665